File: app/src/main/java/com/example/whattowatch/database/AppDatabase.java
package com.example.whattowatch.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.whattowatch.model.mymodel.MyDetailModel;
import com.example.whattowatch.model.mymodel.MyMovieModel;
import com.example.whattowatch.model.mymodel.MyVideoModel;

@Database(entities = { MyMovieModel.class, MyDetailModel.class},
        version = 2)

public abstract class AppDatabase extends RoomDatabase {

    public abstract MovieDao localMovieDAO();
}

File: app/src/main/java/com/example/whattowatch/database/AppDatabase.kt
package com.example.whattowatch.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.example.whattowatch.model.mymodel.MyDetailModel
import com.example.whattowatch.model.mymodel.MyMovieModel

@Database(entities = [MyMovieModel::class, MyDetailModel::class], version = 2)
abstract class AppDatabase : RoomDatabase() {
    abstract fun localMovieDAO(): MovieDao
}